CJBCheatsMenu/i18n/default.json

```diff
@@ -236,6 +236,7 @@
     ** Controls tab
     *********/
     "controls.title": "Controls",
+    "controls.android": "The virtual keys can be chosen by editing this mod's config file.",
```

Comment:
Virtual Keyboard's name is a bit of a misnomer. Instead of a full keyboard, the player gets a series of specific buttons that emulate keyboard or mouse input. The player has to configure them through VK's `config.json` to work with whatever mods they want to use, beyond a base set provided (which include "P" and "I" for the default Cheats Menu and Item Spawner bindings). [Here's the VK interface fully expanded, with my config adjusted to add an F11 button.](https://user-images.githubusercontent.com/4185176/79079189-7401dc00-7cc2-11ea-8979-705f293c1e57.png) If the player were able to tap Set and didn't have a button ready that they wanted to use, they would be trapped in the KeyboardListener overlay or would have to double-assign one of their other keys.

Reply:
Do you mean Virtual Keyboard when you say "this mod" here? If so, maybe we can use something like this to avoid ambiguity:
> You can change the on-screen buttons in the Virtual Keyboard mod's config file.

Reply:
"This mod" is Cheats Menu, as they'll have to edit its `config.json` to set the keybinding itself, but then yes, they would also have to edit VK's `config.json`. Updated to reflect it being both.